os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -491,7 +507,7 @@ void dump_all_stack(void)
  * Name: up_assert
  ****************************************************************************/
 
-void up_assert(const uint8_t *filename, int lineno)
+int up_assert(const uint8_t *filename, int lineno)
```

Comment:
I don't know why is needed? You only return OK

Reply:
yes done with PR #3017